src/components/accounts/UtxoAdapter.ts

```diff
@@ -0,0 +1,47 @@
+import { LocalUtxo, TransactionDetails } from "bdk-rn/lib/classes/Bindings";
+import { Network } from "bdk-rn/lib/lib/enums";
+
+import getAddress from "../shared/getAddress";
+import { Keychain, Utxo } from "../../models/Utxo";
+
+export class UtxoAdapter {
```

Comment:
Not sure about using a class 🤔

Reply:
I guess functions will seem more standard in react, I updated to `toTransaction` and `toUtxo`.  I'll leave them in same directory for now but we'll later want to move them to wherever the contexts go.